src/utils/calendar.ts

```diff
@@ -0,0 +1,44 @@
+import ical from "ical-generator";
+
+import { VenueEvent } from "types/venues";
+
+import { eventEndTime, eventStartTime } from "utils/event";
+
+import { WithVenueId } from "utils/id";
+
+import { getFullVenueInsideUrl } from "utils/url";
```

Comment:
I thought my description was probably going to end up too verbose/complex to be understandable 😛 My intent was to say that we don't need the newlines between these individual imports (just above/below the block of `utils/url` imports as a whole):

```suggestion
import { eventEndTime, eventStartTime } from "utils/event";
import { WithVenueId } from "utils/id";
import { getFullVenueInsideUrl } from "utils/url";
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1423/commits/1346ad92bc3006ba3bbbe1f5695c2806dcc4f4c4